### ISSUE TITLE
Update documentation to explain how to remove transition fields.

### ISSUE THREAD

**Issue by drrobotnik** [Netflix-Skunkworks/go-jira#113]
When using field transitions/in-progress/done, I always get a YAML error. running with the flag -t debug I see:

`updated: Field XXXXXX' cannot be set. It is not on the appropriate screen, or unknown..`

From the documentation it's not clear to me how to exclude a field from the transition templates.

**Comment by coryb** [comment]
Hmm, I am not sure.  Is it possible you are using an old template?  Perhaps you exported it to `$HOME/.jira.d/templates`?  The default template is here:
https://github.com/Netflix-Skunkworks/go-jira/blob/master/jiracli/templates.go#L458
The template first checks the transition metadata to see if a field is allowed in the transition before it  adds it to the document.  Not sure how non-permitted fields would get through that.

Perhaps you are not using a recent `jira`?  
-Cory

**Comment by drrobotnik** [comment]
Okay, tested more, to clarify: I was using jira in-progress XXXXX, which I thought was an alias to transition. I modified my approach to use `jira transition "In Progress" XXXXX` and the YAML syntax error offered the ability to edit. From here I could see that the primary issue was the formatting of our description included multiple hardline breaks and quotes. I needed to add quotes around the description text and escape the content where quotes were found. I was able to get the Story to transition, but it seems like a lot of work each time.

Is it possible to add escaping to all inputs and quotations around text content to the default template?

We're using jira v7.2.8
